src/inputs/UploadFile/UploadFile.js

```diff
@@ -157,7 +157,7 @@ UploadFile.propTypes = {
   /**
    * Function bound on "download" icon
    */
-  handleDownloadFile: PropTypes.func.isRequired,
+  handleDownloadFile: PropTypes.func,
```

Comment:
This change should probably be in a different commit, because it's not related to the change of component style mentioned in the commit message (no need to create a specific branch, you can event push it directly on _main_ :smile: )